tasks/copy_to_glacier_cumulus_translator/test/unit_tests/test_copy_to_glacier_cumulus_translator_unit.py

```diff
@@ -0,0 +1,304 @@
+    def test_handler_and_task_back_and_forth(self, mock_setMetadata: MagicMock):
+        """
+        Tests a happy-path with task to make sure CMA doesn't raise issues.
+        """
+        expected_config = {
+            "file_mapping": {
+                "name": "nameKey",
+                "filepath": "pathKey",
+                "bucket": "bucketKey",
+                "filename": "filenameKey",
+            }
+        }
+        expected_input = {
+            "granules": [
+                {
+                    "granuleId": "some_granule_id",
+                    "files": [
+                        {
+                            "nameKey": "someName",
+                            "pathKey": "somePath",
+                            "bucketKey": "someBucket",
+                            "filenameKey": "someFilename",
+                        }
+                    ],
+                }
+            ]
+        }
+        event = {"task_config": expected_config, "payload": expected_input}
+        context = Mock()
+
+        result = copy_to_glacier_cumulus_translator.handler(event, context)
+        self.assertEqual(
+            {
+                "granules": [
+                    {
+                        "files": [
+                            {
+                                "bucket": "someBucket",
+                                "filename": "someFilename",
+                                "filepath": "somePath",
+                                "name": "someName",
+                            }
+                        ],
+                        "granuleId": "some_granule_id",
+                    }
+                ]
+            },
+            result["payload"],
+        )
+
+    # noinspection PyPep8Naming
+    @patch("cumulus_logger.CumulusLogger.setMetadata")
+    def test_handler_rejects_bad_input(self, mock_setMetadata: MagicMock):
+        expected_config = {
+            "file_mapping": {
+                "name": "nameKey",
+                "filepath": "pathKey",
+                "bucket": "bucketKey",
+                "filename": "filenameKey",
+            }
+        }
+        expected_input = {
+            "granules": [
+                {
+                    "granule_id": "some_granule_id",  # this is the wrong format. Should be "granuleId"
+                    "files": [
+                        {
+                            "nameKey": "someName",
+                            "pathKey": "somePath",
+                            "bucketKey": "someBucket",
+                            "filenameKey": "someFilename",
+                        }
+                    ],
+                }
+            ]
+        }
+        event = {"task_config": expected_config, "payload": expected_input}
+        context = Mock()
+
+        with self.assertRaises(jsonschema.exceptions.ValidationError):
+            copy_to_glacier_cumulus_translator.handler(event, context)
+
+    def test_task_happy_path(self):
```

Comment:
Quick doc string on test would be good here.

Reply:
Added